ArduCopter/ArduCopter.cpp

```diff
@@ -621,7 +621,11 @@ void Copter::read_AHRS(void)
 #endif
 
     // we tell AHRS to skip INS update as we have already done it in fast_loop()
-    ahrs.update(true);
+#if FRAME_CONFIG==ROT_FRAME
```

Comment:
why is this being skipped?

Reply:
because otherwise the standard kalman filter is used and it diverges. I will post a video in a couple of hours that shows it

Reply:
I understand it's because it gets dizzy.. but there's no reason to make a whole new compile option frame type. Can't you just change it EKF_type=0 skips it?